api/env.template

```diff
@@ -4,6 +4,9 @@ DJANGO_SETTINGS_MODULE="catalog.settings"
 DJANGO_SECRET_KEY="ny#b__$$f6ry4wy8oxre97&-68u_0lk3gw(z=d40_dxey3zw0v1"
 DJANGO_DEBUG_ENABLED="True"
 
+# List of comma-separated hosts/domain names, e.g., "127.17.0.1,local.app"
+ALLOWED_HOSTS=""
```

Comment:
Let's add the docker ones to the default so that this works locally without any fuss for new folks.

```suggestion
ALLOWED_HOSTS="172.17.0.1,host.docker.internal"
```